app/parsers/event_parser.rb

```diff
@@ -0,0 +1,53 @@
+class EventParser
+  MissingMeetupDate = Class.new(StandardError)
+
+  def initialize(data)
+    @data = ActiveSupport::HashWithIndifferentAccess.new(data)
```

Comment:
What class is `data`? If it's a `Hash`, ActiveSupport extends that class to declare [`#with_indifferent_access`](https://api.rubyonrails.org/classes/Hash.html#method-i-with_indifferent_access).